frontend/src/app/utils/utils.ts

```diff
@@ -1 +1,2 @@
+export const generateId = () => crypto.randomUUID();
```

Comment:
Is there a reason to have 2 identical functions with different names?

Reply:
during the manual merge with a rebase, I left that over.